### PR TITLE
CLUE 10 cards + 1 MKM

### PR DESCRIPTION
**Pull request by Simisays**
Seems like the MKM was accidently set as merged

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/amzu_swarms_hunger.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Types:Legendary Creature Insect Shaman
+PT:3/3
+K:Flying
+K:Menace
+S:Mode$ Continuous | Affected$ Insect.YouCtrl+Other | AddKeyword$ Menace | Description$ Other Insects you control have menace.
+T:Mode$ ChangesZoneAll | ValidCards$ Card.YouOwn | Origin$ Graveyard | ResolvedLimit$ 1 | Destination$ Any | TriggerZones$ Battlefield | Execute$ TrigToken | TriggerDescription$ Whenever one or more cards leave your graveyard, you may create a 1/1 black and green Insect creature token, then put a number of +1/+1 counters on it equal to the greatest mana value among those cards. Do this only once each turn.
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ bg_1_1_insect | RememberTokens$ True | TokenOwner$ You | SubAbility$ DBPutCounter
+SVar:DBPutCounter:DB$ PutCounter | Defined$ Remembered | CounterNum$ X | CounterType$ P1P1 | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:X:TriggeringObjects$GreatestCMC
```
Comment (Simisays) [inline forge-gui/res/cardsfolder/upcoming/amzu_swarms_hunger.txt:12]: This isn't the correct one. Thoughts ?

File: forge-gui/res/cardsfolder/upcoming/amzu_swarms_hunger.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Name:Amzu, Swarm's Hunger
+ManaCost:3 B G
+Types:Legendary Creature Insect Shaman
+PT:3/3
+K:Flying
+K:Menace
+S:Mode$ Continuous | Affected$ Insect.YouCtrl+Other | AddKeyword$ Menace | Description$ Other Insects you control have menace.
+T:Mode$ ChangesZoneAll | ValidCards$ Card.YouOwn | Origin$ Graveyard | ResolvedLimit$ 1 | Destination$ Any | TriggerZones$ Battlefield | Execute$ TrigToken | TriggerDescription$ Whenever one or more cards leave your graveyard, you may create a 1/1 black and green Insect creature token, then put a number of +1/+1 counters on it equal to the greatest mana value among those cards. Do this only once each turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/amzu_swarms_hunger.txt:8]: optional

File: forge-gui/res/cardsfolder/upcoming/amzu_swarms_hunger.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/amzu_swarms_hunger.txt:12]: TriggerObjectsCards

File: forge-gui/res/cardsfolder/upcoming/covetous_elegy.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Name:Covetous Elegy
+ManaCost:4 W B
+Types:Sorcery
+A:SP$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ DBChooseSac | SubAbility$ DBSac | SpellDescription$ Each player chooses up to two creatures they control, then sacrifices the rest. Then you create a tapped Treasure token for each creature your opponents control.
+SVar:DBChooseSac:DB$ ChooseCard | Defined$ Remembered | Choices$ Creature.RememberedPlayerCtrl | Amount$ 2 | ChoiceTitle$ Choose two creatures not to be sacrificed | RememberChosen$ True | Mandatory$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/covetous_elegy.txt:5]: it's not mandatory

File: forge-gui/res/cardsfolder/upcoming/forum_familiar.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Name:Forum Familiar
+ManaCost:W
+Types:Creature Cat
+PT:1/1
+K:Disguise:1 W
+T:Mode$ TurnFaceUp | ValidCard$ Card.Self | Execute$ TrigChangeZone | TriggerZones$ Battlefield | TriggerDescription$ When CARDNAME is turned face up, return another target permanent you control to its owner's hand and put a +1/+1 counter on Forum Familiar.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/forum_familiar.txt:6]: CARDNAME

File: forge-gui/res/cardsfolder/upcoming/frenzied_gorespawn.txt [hunk 5]
```diff
@@ -0,0 +1,9 @@
+Name:Frenzied Gorespawn
+ManaCost:3 B R
+Types:Creature Horror
+PT:4/4
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigGoad | TriggerDescription$When CARDNAME enters the battlefield, for each opponent, goad target creature that player controls. (Until your next turn, those creatures attack each combat if able and attack a player other than you if able.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/frenzied_gorespawn.txt:5]: spacing

File: forge-gui/res/cardsfolder/upcoming/mastermind_plum.txt [hunk 6]
```diff
@@ -0,0 +1,14 @@
+Name:Mastermind Plum
+ManaCost:2 B
+Types:Legendary Creature Human Wizard
+PT:2/2
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigChangeZone | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME attacks, exile up to one target card from a graveyard. If an artifact card was exiled this way, create a Treasure token. (It's an artifact with "{T}, Sacrifice this artifact: Add one mana of any color.")
+SVar:TrigChangeZone:DB$ ChangeZone | Origin$ Graveyard | TargetMin$ 0 | TargetMax$ 1 | Destination$ Exile | ValidTgts$ Card | TgtPrompt$ Select up to one target card in a graveyard | RememberChanged$ True | SubAbility$ DBToken
+SVar:DBToken:DB$ Token | TokenAmount$ 1 | TokenScript$ c_a_treasure_sac |ConditionDefined$ Remembered | ConditionPresent$ Creature | ConditionCompare$ EQ0 | SubAbility$ DBCleanup
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/mastermind_plum.txt:7]: bad condition

File: forge-gui/res/cardsfolder/upcoming/frenzied_gorespawn.txt [hunk 7]
```diff
@@ -0,0 +1,9 @@
+Name:Frenzied Gorespawn
+ManaCost:3 B R
+Types:Creature Horror
+PT:4/4
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigGoad | TriggerDescription$ When CARDNAME enters the battlefield, for each opponent, goad target creature that player controls. (Until your next turn, those creatures attack each combat if able and attack a player other than you if able.)
+SVar:TrigGoad:DB$ Goad | TargetMin$ 0 | TargetMax$ X | ValidTgts$ Creature.OppCtrl | TargetsWithDifferentControllers$ True | TgtPrompt$ Goad up to one target creature that opponent controls
+T:Mode$ AttackersDeclared | ValidCard$ Creature | AttackedTarget$ Opponent | Execute$ TrigPumpAll | TriggerZones$ Battlefield | TriggerDescription$ Whenever one or more creatures attack one of your opponents, those creatures gain menace until end of turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/frenzied_gorespawn.txt:7]: wrong triggertype

File: forge-gui/res/cardsfolder/upcoming/frenzied_gorespawn.txt [hunk 8]
```diff
@@ -0,0 +1,9 @@
+Name:Frenzied Gorespawn
+ManaCost:3 B R
+Types:Creature Horror
+PT:4/4
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigGoad | TriggerDescription$ When CARDNAME enters the battlefield, for each opponent, goad target creature that player controls. (Until your next turn, those creatures attack each combat if able and attack a player other than you if able.)
+SVar:TrigGoad:DB$ Goad | TargetMin$ 0 | TargetMax$ X | ValidTgts$ Creature.OppCtrl | TargetsWithDifferentControllers$ True | TgtPrompt$ Goad up to one target creature that opponent controls
+T:Mode$ AttackersDeclaredOneTarget | ValidAttackers$ Creature | AttackedTarget$ Opponent | Execute$ TrigPumpAll | TriggerZones$ Battlefield | TriggerDescription$ Whenever one or more creatures attack one of your opponents, those creatures gain menace until end of turn.
+SVar:TrigPumpAll:DB$ PumpAll | ValidCards$ Creature.attackingOpponent | KW$ Menace
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/frenzied_gorespawn.txt:8]: you're only correct if pumping TriggeredAttackers

**Review by tool4ever — APPROVED** [review]

**Pull request merged**